evaluation/evaluation.go

```diff
@@ -101,8 +102,10 @@ func (e *evaluator) EvaluateFeaturesByEvaluatedAt(
 	if len(updatedFeatures) == 0 {
 		return e.evaluate(fs, user, mapSegmentUsers, true, targetTag)
 	}
-	featuresHavePrerequisite := e.getFeaturesHavePrerequisite(fs)
-	evalTargets := e.getPrerequisiteUpwards(updatedFeatures, featuresHavePrerequisite)
+	evalTargets, err := e.getEvalFeatures(updatedFeatures, fs)
```

Comment:
We need to add this before calling `evaluate` here, too.

https://github.com/bucketeer-io/bucketeer/blob/16f9c2fc693eba6749ce753cab8bf78a44b475c8/evaluation/evaluation.go#L70

Reply:
The `GetEvaluation` API call this [getTargetFeatures](https://github.com/bucketeer-io/bucketeer/blob/16f9c2fc693eba6749ce753cab8bf78a44b475c8/pkg/gateway/api/api_grpc.go#L924-L934), which calls the `GetPrerequisiteDownwards`.

Reply:
For this case, don't we also need to call the `getFeaturesDependedOnTargets`?

Reply:
It is not necessary to be added here because `fs` includes all features to be evaluated 👍

Reply:
I modified `GetPrerequisiteDownwards()` to call `getFeaturesDependedByTargets()`.